docs/SUMMARY.md

```diff
@@ -88,19 +88,158 @@
   - [OpenTracing](opentracing.md)
   - [Database Schemas](development/database_schema.md)
   - [Experimental features](development/experimental_features.md)
-  - [Synapse Architecture]()
-    - [Log Contexts](log_contexts.md)
-    - [Replication](replication.md)
-    - [TCP Replication](tcp_replication.md)
-  - [Internal Documentation](development/internal_documentation/README.md)
+  - [Synapse Architecture](development/synapse_architechture.md)
+    - [Overview]()
+      - Flows of data through Synapse
+        - Starts with a Request/BG Job/Module
+        - Processing
+        - Maybe Response
+      - Config
+      - Request Handling
+        - Clients
+        - Federation
+        - Internal Processing - there could be an exception!
+      - Storage
+      - Federation
+      - External Entities: Remote homeservers, Appservices, Pushers, Identity Servers
+      - Modules
+      - Workers/Replication/Redis/Organization of a large homeserver instance
+  - [Configuration]()
+    - Experimental config options
+      - Define how long they should live after the corresponding msc has merged
+  - [Logging]()
+    - combine with [Log Contexts](log_contexts.md)
+  - [Storage]()
+    - What is current database schema?
+      - Start by linking script that can compile full schema
+      - Is there a tool that can visualize a postgres db? - Shay to check
```

Comment:
I like this section 👍

Some copy-pastable snippets on how to explore the database would also be nice (cheat sheet):

>  - `psql synapse`: Open the postgres CLI against the `synapse` database
>  - `\q` -> `enter`: To quit/exit
>  - `\dt`: list tables
>  - `\d+ issues`: list columns and indexes for issues table
>  - `CREATE TABLE board_labels();`: Create a table called board_labels
>  - `DROP TABLE insertion_event_extremeties`
>  - `explain SELECT * FROM events WHERE depth = 1;`
>  - `explain analyze SELECT * FROM events WHERE depth = 1;`
>
> *-- https://github.com/matrix-org/synapse/pull/10245*